charts/victoria-metrics-agent/templates/service-monitor.yaml

```
@@ -1,48 +1,44 @@
 {{- if .Values.serviceMonitor.enabled -}}
+{{- $serviceMonitor := .Values.serviceMonitor -}}
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
-  {{- if .Values.serviceMonitor.annotations }}
-  annotations:
-{{ toYaml .Values.serviceMonitor.annotations | indent 4 }}
-  {{- end }}
-  labels:
-  {{- include "chart.labels" . | nindent 4 }}
-  {{- with .Values.serviceMonitor.extraLabels -}}
-  {{ toYaml . | nindent 4 }}
+  {{- with $serviceMonitor.annotations }}
+  annotations: {{ toYaml $serviceMonitor.annotations | nindent 4 }}
```

As far as I can see, that should be:
```suggestion
  annotations: {{ toYaml . | nindent 4 }}
```